test/unit/range/views/as_const_test.cpp

```diff
@@ -15,9 +15,15 @@
 #include <seqan3/alphabet/views/complement.hpp>
 #include <seqan3/core/detail/debug_stream_alphabet.hpp>
 #include <seqan3/range/concept.hpp>
+#ifdef SEQAN3_DEPRECATED_310
```

Comment:
Why?

Reply:
True, might not be necessary xD

My thought was: If we have api stability tests of seqan 3.1.0 with seqan 3.0.3 test, I wouldn't need to add a patch to remove this. But on second thought, I'll most definitely just cherry-pick the commit that removes all deprecated code in seqan3.1.